zulip_bots/zulip_bots/bots/followup/fixtures/test_send_reply.json

```diff
@@ -0,0 +1,8 @@
+{
+    "expectations":[
+        {
+            "request":"",
+            "response":"Please specify the message you want to send to followup stream after @mention-bot"
+        }
+    ]
+}
```

Comment:
I think the linter is probably not running against the JSON files?  We do that in the server repository, but maybe that bit of config wasn't ported over.